app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MoviesController < ApplicationController
+
+  def show
+    @movie = Movie.find(params[:id])
+  end
+
+  def update
```

Comment:
This implementation looks good but a more RESTful routing option here would be to route this form to a joins contoller like `MovieActorsController #create` to indicate that you are creating an association between an existing movie and an actor. We want to reserve the movies#update action for updating attributes on a movie resource itself.